app.js

```diff
@@ -0,0 +1,22 @@
+const express = require("express");
+const app = express();
+// app.use(express.json());
+
+const {
+    getTopics
+} = require('./controllers/topics.controller')
```

Comment:
things being required in should go at the top of the file, so in this case before line 2 :)